.cbmc-batch/jobs/aws_cryptosdk_sig_sign_start/aws_cryptosdk_sig_sign_start_harness.c

```diff
@@ -23,29 +23,29 @@
 #include <cipher_openssl.h>
 
 void aws_cryptosdk_sig_sign_start_harness() {
-    /* arguments */
+    /* Nondet Inputs */
     struct aws_cryptosdk_sig_ctx *ctx;
     struct aws_allocator *alloc = can_fail_allocator();
     struct aws_string *pub_key;
     struct aws_string *priv_key = ensure_string_is_allocated_nondet_length();
     enum aws_cryptosdk_alg_id alg_id;
     struct aws_cryptosdk_alg_properties *props = aws_cryptosdk_alg_props(alg_id);
 
-    /* assumptions */
-    __CPROVER_assume(props);
-    assert(aws_string_is_valid(priv_key));
+    /* Assumptions */
+    __CPROVER_assume(aws_cryptosdk_alg_properties_is_valid(NULL));
+    __CPROVER_assume(aws_string_is_valid(NULL));
 
     bool save_pub_key = nondet_bool();
 
-    /* operation under verification */
+    /* Operation under verification */
     if (aws_cryptosdk_sig_sign_start(
             &ctx, alloc, save_pub_key ? &pub_key : NULL, props, priv_key /* priv_key can't be NULL */) ==
```

Comment:
`&ctx` is never null. Is this what you intend?

Reply:
ctx itself can be NULL, I've updated the harness. Its address shouldn't be.